server/odc-service/src/main/java/com/oceanbase/odc/service/iam/model/JwtProperties.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.iam.model;
+
+public class JwtProperties {
+
+    public static final String ID = "id";
+    public static final String TOKEN = "token";
```

Comment:
the name 'token' may too general here.  may use 'odcJwtToken'.

another point, here ID, TOKEN concept seems not JWT properties itself.

Reply:
The token name is changed to 'odcJwtToken'. 
On another point, jwt carries default payloads, which can also be customized.